src/v/kafka/server/handlers/api_versions.cc

```diff
@@ -71,8 +71,51 @@ serialize_apis(type_list<RequestTypes...>) {
     return apis;
 }
 
+static std::vector<api_versions_response_key>
```

Comment:
If this is on the hot path then we should do 3 static lists computed apriorí and simply if-return

alternatively you can compute them statically with a fold-expression I think

Reply:
c++ meta programming is above me I went for 3 static lists